chain/network/src/peer_manager/tests/nonce.rs

```diff
@@ -2,11 +2,14 @@ use crate::network_protocol::testonly as data;
 use crate::network_protocol::{
     Encoding, Handshake, PartialEdgeInfo, PeerMessage, EDGE_MIN_TIMESTAMP_NONCE,
 };
-use crate::peer_manager;
+use crate::peer_manager::testonly::{ActorHandler, Event};
+use crate::peer_manager::{self, peer_manager_actor};
 use crate::tcp;
 use crate::testonly::make_rng;
 use crate::testonly::stream;
 use crate::time;
+use crate::types::Edge;
+use ::time::Duration;
```

Comment:
```use crate::time;```
don't import Duration directly, and use crate::time instead of ::time.

Reply:
done.